include/faker-cxx/Book.h

```diff
@@ -61,5 +61,68 @@ class Book
      * @endcode
      */
     static std::string isbn();
+
+	/**
+	 * @brief Returns a random release year
+	 *
+	 * @returns int year
+	 *
+	 * @code
+	 * Book::releaseYear() // "2016-06-27"
+	 * @endcode
+	 */
+	static std::string releaseYear();
+
+	/**
+	 * @brief Returns the full name of a translator
+	 *
+	 * @returns std::string full name
+	 *
+	 * @code
+	 * Book::translator() // "Eric Floyd"
+	 * @endcode
+	 */
+	static std::string translator();
+	
+	enum class BookFormat {
+		Paperback,
+		Hardcover,
+		Kindle,
+		// Add more formats if needed
+	};
+	
+	/**
+	 * @brief Returns format of book
+	 *
+	 * @returns BookFormat format of book
+	 *
+	 * @code
+	 * Book::format() // BookFormat::paperback
+	 * @endcode
+	 */
+	static BookFormat format();
```

Comment:
I am thinking that I may be better to return them as string and not define this enum at all because for end user it would be easier to use string and we can write in function docs that it has values of (paperback etc).

Reply:
Yeah in the issue description it said "format (one of values: paperback, hardcover, kindle or even more...)" I assumed "one of values" meant enum. I'll fix it.